qemu/tests/cfg/boot_order_check.cfg

```diff
@@ -27,6 +27,7 @@
     force_create_image_stg2 = yes
     remove_image_stg2 = yes
     nic_addr_filter = "%s.*?Bus\s+(\d+),\s+device\s+(\d+),\s+function\s+(\d+)"
+    cdroms = ''
```

Comment:
Please leave a comment for this line to describe the issue you met, thanks.

Reply:
Updated, please help review again, thanks! @luckyh